chartfx-samples/src/main/java/de/gsi/dataset/samples/DataSetToByteArraySample.java

```diff
@@ -142,7 +146,11 @@ public void testGenericSerializerPerformance(final int iterations, final boolean
             serialiser.serialiseObject(dsOrig);
 
             byteBuffer.reset(); // reset to read position (==0)
-            serialiser.deserialiseObject(cpOrig);
+            final Object retOrig = serialiser.deserialiseObject(cpOrig);
+
+            if (retOrig != cpOrig) {
```

Comment:
Codacy found an issue: [Use equals() to compare object references.](https://app.codacy.com/gh/GSI-CS-CO/chart-fx/pullRequest?prid=6382991)